### PR TITLE
Describing the new option to restore snapshot in onboarding

### PR DESCRIPTION
**Pull request by Amir974**
Adding description to the new onboarding feature.
## Proposed change

Updated image with link allowing to upload snapshot

a note at the top explaining the 2 options (configure fresh setup / upload snapshot,
a new paragraph at the bottom (following the configuration process) describing the alternative route of uploading a snapshot.

Can I reuse the image from the blog post / do you think I should add more images or details?

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/getting-started/onboarding.markdown [hunk 1]
```diff
@@ -5,10 +5,14 @@ description: "Instructions to get Home Assistant configured."
 
 Alright, you made it here. The tough part is done.
 
-With Home Assistant installed, it's time to configure it. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
+With Home Assistant installed, it's time to configure it. 
+
+If this is your first time installing Home Assistant, or if you want a fresh install keep reading. However, if you have a snapshot saved of a previous setup that you want to load, this is your chance to do so using the link at the bottom - scroll down for the details.
```
Comment (frenck) [inline source/getting-started/onboarding.markdown:10]: I feel like we are out of context here.

The getting started guide, is aimed at getting started for the first time.
Considering that, putting in a restore guide in here, feels a bit out of place?

Comment (Amir974) [reply source/getting-started/onboarding.markdown:10]: I see your point, but in reality I found myself going back to this flow (which is one click a away from homepage rather then going to the documentation and finding the relevant page).

I don’t think it hurts to have a paragraph for people looking to recover a previous version - think of it as “getting started (all over again)” ;)

Comment (Amir974) [reply source/getting-started/onboarding.markdown:10]: I guess no answer means you don’t see it that way?
* Is there anything you want to keep from the above changes?
* Would you like to have a dedicated page for restoring upon starting over ?
* if so, can we have a link from this page to that new on or would that also feel out of place in your opinion?

@frenck

**Comment by Amir974** [comment]
Lost interest

File: source/getting-started/onboarding.markdown (hunk 1 above)
Comment (frenck) [reply source/getting-started/onboarding.markdown:10]: No answer means, we are very busy with over 600 Prs open and receiving around 100-150 new PRs each day. It means, have some patience.

**Pull request closed without merging**